### PR TITLE
Availability event can be used with different status, durations and probabilites

### PR DESCRIPTION
**Pull request by scfx**
Also added sim 11 to test functionality. Haven't found an API to upload the calculation categories to c8y. If there is one, i can also add the category upload. Currently both calculation categories: "Manual stop" & "Planned maintenance" need to be added manually in the UI.
image

### CODE REVIEW

**Comment by thomaswinkler** [comment]
@scfx 

To upload the calculation categories we just need to create a managed object with type ```OEECategoryConfiguration``` and ```categories``` assigned as custom properties.

``` JSON
"type": "OEECategoryConfiguration",
"categories": [
      {
         "categoryInput": "Actual Production Amount",
         "name": "Test",
         "id": "1",
         "status": "ACTIVE"
     }
]
```

Would be great if this could be also automated.

**Comment by haraldmeyer** [comment]
@scfx @thomaswinkler though you have to be careful. If there exists one such MO, you have to update it instead of creating a new one, as we expect there to be only one.

**Review by haraldmeyer — APPROVED** [review]

File: event-based-simulators/.vscode/launch.json [hunk 1]
```diff
@@ -1,53 +1,77 @@
-{
```
Comment (haraldmeyer) [inline event-based-simulators/.vscode/launch.json:1]: Is this file required in the repository?

**Review by mvoigt-sag — CHANGES_REQUESTED** [review]
looks good so far, but I'd prefer to have a dedicated option to enable/disable the creation of categories and not (mis)use the CREATE_PROFILES property

File: event-based-simulators/main/event_based_simulators.py [hunk 2]
```diff
@@ -358,6 +371,7 @@ def load(filename):
 if CREATE_PROFILES.lower() == "true":
     [oeeAPI.create_and_activate_profile(id, ProfileCreateMode.CREATE_IF_NOT_EXISTS) 
         for id in oeeAPI.get_simulator_external_ids()]
+    os.system("python profile_generator.py -cat")
```
[the remark this answers is not in the document]
Comment (haraldmeyer) [reply event-based-simulators/main/event_based_simulators.py:374]: @mvoigt-sag I am not sure about that. One of the profiles requires the presence of the categories. So creating one without the other does not make much sense.

**Review by haraldmeyer — APPROVED** [review]

File: event-based-simulators/main/event_based_simulators.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (thomaswinkler) [reply event-based-simulators/main/event_based_simulators.py:374]: Currently we are using ids 1 and 2. Looks like current implementation takes care of POST vs PUT the categories, but if there are categories already the might be overwritten, correct?

[the remark this answers is not in the document]
Comment (scfx) [reply event-based-simulators/main/event_based_simulators.py:374]: Yes, its checked if the managed object of that type exists and in case it exists it will be overwritten. If there are more than one object of that type, it will log an error.

[the remark this answers is not in the document]
Comment (mvoigt-sag) [reply event-based-simulators/main/event_based_simulators.py:374]: OK, so we should maybe use other IDs for the categories. maybe some higher numbers like 10001 & 10002.

**Comment by haraldmeyer** [comment]
WIll merge as there is new PR #19
